core/src/input/input.stories.tsx

```diff
@@ -2,50 +2,47 @@ import { useState } from "react";
 import { Button } from "../button/button";
 import { Dialog } from "../dialog/dialog";
 import { DivPx } from "../div/div";
+import { FormField } from "../form/field";
 import { _Story } from "../_story";
 import { Input } from "./input";
+import { HiPhone } from "react-icons/hi";
 
 export default {
-	title: "Draft/Input",
+	title: "Components/Input",
 	component: Input,
 	argTypes: {
-		type: _Story.arg([
-			"text",
-			"number",
-			"email",
-			"password",
-			"url",
-			"color",
-		]),
-		style: _Story.arg(Input.styles),
-		size: _Story.arg(Input.sizes),
-		maxLength: _Story.arg("number"),
-		disabled: _Story.arg("boolean"),
-		readOnly: _Story.arg("boolean"),
-		placeholder: _Story.arg(null),
-		icon: _Story.arg(null),
-		defaultValue: _Story.arg(null),
-		forwardedRef: _Story.arg(null),
-
-		value: _Story.arg(null),
-		setValue: _Story.arg(null),
-
-		list: _Story.arg(null),
-
-		id: _Story.arg(null),
-		autoFocus: _Story.arg(null),
-		autoSelect: _Story.arg(null),
-		"aria-label": _Story.arg(null),
-		"aria-labelledby": _Story.arg(null),
-
-		onBlur: _Story.arg(null),
-		onFocus: _Story.arg(null),
-		onKeyPress: _Story.arg(null),
-		onKeyUp: _Story.arg(null),
-		onKeyDown: _Story.arg(null),
-		onClick: _Story.arg(null),
-		onChange: _Story.arg(null),
+		type: _Story.arg(
+			["text", "number", "email", "password", "url", "color"],
+			"Visual"
+		),
+		style: _Story.arg(Input.styles, "Visual"),
+		size: _Story.arg(Input.sizes, "Visual"),
+		placeholder: _Story.arg(null, "Visual"),
+		icon: _Story.arg(null, "Visual"),
+		defaultValue: _Story.arg(null, "Uncontrolled"),
+		value: _Story.arg(null, "Controlled"),
+		setValue: _Story.arg(null, "Controlled"),
+		list: _Story.arg(null, "Controlled"),
```

Comment:
I don't think "list" should be a part of the "Controlled" group. It also works in uncontrolled input. Should move it to the "Attributes" group.